### PR TITLE
Debugger - show toggle button and toolbar

### PR DESCRIPTION
**Pull request by gushuro**
Fixing error causing debug toggle button and toolbar not to be shown.

### CODE REVIEW

File: webapp/src/app.tsx [hunk 1]
```diff
@@ -2729,7 +2729,7 @@ export class ProjectView
         const inHome = this.state.home && !sandbox;
         const inEditor = !!this.state.header && !inHome;
         const { lightbox, greenScreen } = this.state;
-        const simDebug = !!targetTheme.debugger;
+        const simDebug = pxt.editor.experiments.isEnabledByKey('experiments-debugger');
```
Comment (pelikhan) [inline webapp/src/app.tsx]: I don’t understand this change.

File: pxteditor/experiments.ts [hunk 2]
```diff
@@ -96,6 +96,10 @@ namespace pxt.editor.experiments {
         return all().some(experiment => isEnabled(experiment));
     }
 
+    export function isEnabledByKey(experimentKey: string): boolean {
```
Comment (pelikhan) [inline pxteditor/experiments.ts]: You are leaking the internal key representation for experiencents. This should not happen.

File: webapp/src/editortoolbar.tsx [hunk 3]
```diff
@@ -134,13 +134,13 @@ export class EditorToolbar extends data.Component<ISettingsProps, {}> {
         const trace = !!targetTheme.enableTrace;
         const tracing = this.props.parent.state.tracing;
         const traceTooltip = tracing ? lf("Disable Slow-Mo") : lf("Slow-Mo")
-        const debug = !!targetTheme.debugger && !readOnly;
+        const debug = !!pxt.editor.experiments.isEnabledByKey('experiments-debugger') && !readOnly;
```
Comment (pelikhan) [inline webapp/src/editortoolbar.tsx]: Revert this too.

**Comment by gushuro** [comment]
Closing as this was fixed in https://github.com/Microsoft/pxt/pull/5326

**Pull request closed without merging**